Formula/boxes.rb

```diff
@@ -1,9 +1,9 @@
 class Boxes < Formula
   desc "Draw boxes around text"
   homepage "https://boxes.thomasjensen.com/"
-  url "https://github.com/ascii-boxes/boxes/archive/v2.1.1.tar.gz"
-  sha256 "95ae6b46e057a79c6414b8c0b5b561c3e9d886ab8123a4085d256edccce625f9"
-  license "GPL-2.0-only"
+  url "https://github.com/ascii-boxes/boxes/archive/v2.2.0.tar.gz"
+  sha256 "98b8e3cf5008f46f096d5775d129c34db9f718728bffb0f5d67ae89bb494102e"
+  license "GPL-3.0"
```

Comment:
`-only` or `-or-later`?

Reply:
I picked what the CI gave, how is the difference done and which one should be picked when the license file just give GPL v3?

Reply:
Check the source files, they should indicate the restrictions on the license

Reply:
I don't see any `later` so I guess that means it's only? https://github.com/ascii-boxes/boxes/blob/master/LICENSE

Reply:
Yea, seems like `-only`. https://github.com/ascii-boxes/boxes/blob/d572344fced538d307d8f4eafaafd5b8133ce003/src/boxes.c#L1-L14

Reply:
See also https://github.com/ascii-boxes/boxes/issues/101#issuecomment-1250377136